custom/ewsghana/reports/stock_levels_report.py

```diff
@@ -317,17 +319,24 @@ def rendered_content(self):
             'url': reverse(EditCommCareUserView.urlname, args=[self.config['domain'], sms_user.get_id])
         }
 
+        web_users_from_extension = list(iter_docs(
+            WebUser.get_db(),
+            EWSExtension.objects.filter(domain=self.domain,
+                                        location_id=self.location_id).values_list('user_id', flat=True)
+        ))
+
         web_users = [
             {
                 'id': web_user['_id'],
                 'first_name': web_user['first_name'],
                 'last_name': web_user['last_name'],
-                'email': web_user['email']
+                'email': web_user['username']
             }
-            for web_user in UserES().web_users().domain(self.config['domain']).term(
-                "domain_memberships.location_id", self.config['location_id']
-            ).run().hits
+            for web_user in (UserES().web_users().domain(self.config['domain']).term(
+                "domain_memberships.location_id", self.location_id
+            ).run().hits + web_users_from_extension)
```

Comment:
would the query from es and the result from web_users_from_extension ever overlap? if so we should make the elements in the result be unique